crates/ghost_actor/src/ghost_system.rs

```diff
@@ -16,6 +16,21 @@ impl Default for GhostProcessInstructions {
     }
 }
 
+pub trait GhostSystemRef<'lt> {
```

Comment:
```suggestion
pub trait GhostSystemRef<'lt>: Send + Sync + Clone {
```

If we put the required super-trait bounds here, we can make sure GhostSystemRef s always implement these without needing the bounds everywhere they are used.